app/src/main/java/com/lagradost/quicknovel/MadaraInterface.kt

```diff
@@ -0,0 +1,317 @@
+package com.lagradost.quicknovel
+
+import android.net.Uri
+import java.util.*
+import org.jsoup.Jsoup
+import org.jsoup.Connection
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+
+// nneed help for adding cookies and dns cache
+fun connect(url: String): Connection = Jsoup.connect(url).apply {
```

Comment:
🤨 does khttp not work at all? using Jsoup.connect is very discouraged because I might change to nicehttp soon and using khttp makes the transition easier

Reply:
This also makes code review and maintenance a lot harder because in this project no provider uses Jsoup for any network requests

Reply:
> 🤨 does khttp not work at all? using Jsoup.connect is very discouraged because I might change to nicehttp soon and using khttp makes the transition easier

of course it works fine. it's just that I consider it practical. but if it has to be changed to khttp I think it's okay .. I'll change it.